modules/graphql_api_endpoint/outputs.tf

```diff
@@ -1,3 +1,7 @@
 output "integration_id" {
   value = aws_api_gateway_integration.graphql_api.id
 }
+
+output "relative_path" {
+  value = aws_api_gateway_resource.graphql_endpoint.path
```

Comment:
Minor thing, but this is the complete path rather than the relative path. The relative path of an API Gateway endpoint is relative to its parent resource, but this path will include all the parents.

Reply:
I've changed it to path as the terraform variable is called path, may as well be consistent.